code/datums/components/crafting/guncrafting.dm

```diff
@@ -11,5 +11,6 @@
 /obj/item/weaponcrafting/stock
 	name = "rifle stock"
 	desc = "A classic rifle stock that doubles as a grip, roughly carved out of wood."
+	custom_materials = list(/datum/material/wood = 12000)
```

Comment:
thats a lot of wood

Reply:
It's made from 10 planks of wood originally, you lose 8k in the carving process.